components/Search/SearchResults/index.js

```diff
@@ -0,0 +1,27 @@
+import { interviews } from "@/public/interviews";
+
+export default function SearchResults({ searchTerm }) {
+  const results = interviews.filter((interview) =>
+    interview.tags.some((tag) => tag.includes(searchTerm))
```

Comment:
```suggestion
        interview.tags.some((tag) => tag.toLowerCase().includes(searchTerm.toLowerCase()))
```
If you sanitize the tags and the searchTerm where you check one against the other, it won't matter whether the original inputs/etc were uppercase or lowercase. :)

Reply:
Works like a charm! Thank you.